packages/libclang.runtime.osx-x64/libclang.runtime.osx-x64.nuspec

```diff
@@ -2,18 +2,18 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2013/05/nuspec.xsd">
   <metadata minClientVersion="2.12">
     <id>libclang.runtime.osx-x64</id>
-    <version>8.0.0</version>
+    <version>9.0.0</version>
     <authors>Microsoft and Contributors</authors>
     <owners>Microsoft and Contributors</owners>
     <requireLicenseAcceptance>true</requireLicenseAcceptance>
     <license type="expression">Apache-2.0</license>
     <projectUrl>https://github.com/microsoft/clangsharp</projectUrl>
     <description>osx x64 native library for libclang.</description>
     <copyright>Copyright © University of Illinois at Urbana-Champaign</copyright>
-    <repository type="git" url="https://github.com/llvm-mirror/clang" branch="release_80" />
+    <repository type="git" url="https://github.com/llvm/llvm-project" branch="llvmorg-9.0.0" />
```

Comment:
All other rid's now have pre-built versions of `libclang`. macOS is an exception where you still need to build it manually.